tool/lib/webqa-canonical-generator.js

```diff
@@ -20,6 +20,17 @@ const ANNOTATED_PROPERTIES = [
     'geo', 'address.streetAddress', 'address.addressCountry', 'address.addressRegion', 'address.addressLocality'
 ];
 
+
+// turn entity type into string
+function entityTypeToString(type) {
+    if (type.isArray)
+        return entityTypeToString(type.elem);
```

Comment:
It took me an extra second to understand what this function does. The name is not great, because it doesn't turn the type into a string, it find the entity type name out of a type. So maybe `typeToEntityType` or something like that? Or a better comment that doesn't just repeat the function name...